src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+
+  .logo {
+    height: 40px;
```

Comment:
The `.logo` block has a fixed height and width which may cause issues with responsiveness. Consider using more flexible units like percentages or viewport units, or ensure that the logo image is responsive.